dialect/sql/builder.go

```diff
@@ -682,7 +682,7 @@ func (i *InsertBuilder) Returning(columns ...string) *InsertBuilder {
 // Query returns query representation of an `INSERT INTO` statement.
 func (i *InsertBuilder) Query() (string, []interface{}) {
 	i.WriteString("INSERT INTO ")
-	if i.schema != "" {
+	if i.schema != "" && i.dialect != dialect.SQLite {
 		i.Ident(i.schema).WriteByte('.')
 	}
```

Comment:
Maybe moving this code block to the `Builder`, and then calling it from all builders as follows:

```go
func (b *Builder) writeSchema(s schema)  {
	if schema != "" && u.dialect != dialect.SQLite {
		b.Ident(u.schema).WriteByte('.')
	}
}

// ..

u.writeSchema(u.schema)
```

Reply:
@a8m done 👍🏼